### PR TITLE
Fixes github link to healthchecks

### PR DESCRIPTION
**Pull request by Braintelligence**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Image available from:
 * [**Docker Hub**](https://hub.docker.com/r/galexrt/healthchecks)
 
 
-Simple to use Docker image for [https://github.com/healthchecks/healthchecks](healthchecks/healthchecks).
+Simple to use Docker image for [https://github.com/healthchecks/healthchecks](https://github.com/healthchecks/healthchecks).
```
Comment (galexrt) [inline README.md]: Could you please also change the name for the name (in `[]`) to just `github.com/healthchecks/healthchecks` (without https)?

Comment (Braintelligence) [reply README.md]: Like this?

**Comment by galexrt** [comment]
Thank you for your contribution!
Could you go ahead and squash your commits, then I'm happy to merge your changes. :+1:

**Comment by Braintelligence** [comment]
Sorry I didn't do commit squashing yet. I thought it was something that is set up from the base branch side. Is it available withing GitHub or do I have to spin up a git client?

**Comment by galexrt** [comment]
No, it is okay I just saw that I can squash and merge in GitHub directly now.

**Pull request merged**